docs/sql-ref-functions-builtin.md

```diff
@@ -17,202 +17,105 @@ license: |
   limitations under the License.
 ---
 
-{% for static_file in site.static_files %}
-    {% if static_file.name == 'generated-agg-funcs-table.html' %}
+* Table of contents
+{:toc}
```

Comment:
Can we probably have the toc only for the head titles? I think `Examples` subtitle is too verbose.

Reply:
I thought the same thing, but unfortunately the TOC depth can only be set site-wide: https://github.com/toshimaru/jekyll-toc/issues/120

Shall I remove the TOC entirely?

Reply:
Hmmmmmmm.... I wonder if it's better to just remove it for now ....

Reply:
Or .. we can switch `Examples` to just bold instead of h3 title ...

Reply:
not sure which one would be better :-).

Reply:
I'll try the bold and see how that looks. I'll update the screenshot in the PR description, too.

Reply:
Done.